.changeset/cyan-snails-peel.md

```diff
@@ -0,0 +1,12 @@
+---
+'@backstage/plugin-user-settings-backend': patch
+'@backstage/plugin-devtools-backend': patch
+'@backstage/plugin-techdocs-backend': patch
+'@backstage/plugin-catalog-backend': patch
+'@backstage/plugin-search-backend': patch
+'@backstage/plugin-proxy-backend': patch
+'@backstage/plugin-auth-backend': patch
+'@backstage/plugin-app-backend': patch
+---
+
+Migrate the `dev` server to use the new backend system.
```

Comment:
```suggestion
Updated local development setup.
```

Thinking that's even more clear that people shouldn't care? 😁 since this doesn't really affect the published package